socorro/lib/search_common.py

```diff
@@ -375,7 +375,8 @@ def fix_version_parameter(parameters):
         # Iterate on a copy so we can delete elements while looping.
         for version in list(parameters['version']):
```

Comment:
It is not converted, it is copied, so that we can alter the original list while iterating on it.